query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/QueryRequestToPinotSQLConverter.java

```diff
@@ -37,6 +37,9 @@
   private static final int MAP_KEY_INDEX = 0;
   private static final int MAP_VALUE_INDEX = 1;
 
+  private static final List<Operator> SUPPORTED_OPERATORS_FOR_MAP_ATTRIBUTES =
+      List.of(Operator.EQ, Operator.NEQ, Operator.GT, Operator.GE, Operator.LT, Operator.LE);
```

Comment:
Will do in a follow up PR.

Reply:
Some of the boolean operators like and/or/not are not appropriate for map attributes.

Reply:
That's the case for any attribute though, it's not unique to maps. The new commit looks right, but almost all of the other new changes that are are custom to maps should be able to be reverted

Reply:
I think we will not need this set for attribute expression having subpath (so for the key of the map). There is support for `IN`, `NOT IN` as well. So, it will support all the operator ColumnIdentifier was supporting. Can we remove this set? If, any exception that encounter, we can just put that as an exception
e.g
```
select mapValue(tags__KEYS, 'response_flags', tags__VALUES) from spanEventView 
where start_time_millis > 1638497360000 and start_time_millis < 1638508160000
and tags__KEYS = 'response_flags' and mapValue(tags__KEYS, 'response_flags', tags__VALUES) IN ('NR', 'UR')
```

Reply:
resolved